src/mmw/js/src/modeling/models.js

```diff
@@ -10,6 +10,24 @@ var Backbone = require('../../shim/backbone'),
     turfErase = require('turf-erase'),
     turfIntersect = require('turf-intersect');
 
+var GWLFE = 'gwlfe';
+var TR55_TASK = 'tr55';
+var TR55_PACKAGE = 'tr-55';
+
+var modelPackages = [
+    {
+        name: TR55_PACKAGE,
+        displayName: 'Site Storm Model',
+        description: ''
+    },
+    {
+        name: GWLFE,
+        displayName: 'Watershed Multi-Year Model',
+        description: '',
+        disabled: true
```

Comment:
Consider making this a setting somewhere in [`base.py`](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/mmw/settings/base.py), and threading this through via [`window.clientSettings`](https://github.com/WikiWatershed/model-my-watershed/blob/e1a86e60f3bbc0ae26433f891437414129c281cd/src/mmw/apps/core/templates/base.html#L60).